src/main/java/Doug.java

```diff
@@ -0,0 +1,180 @@
+                + "                                     *#***(**///(((((((////*****************%*\n"
+                + "                                     *#*************************************#(\n"
+                + "                                     *#*************************************%/\n"
+                + "                                     ,#*************************************&@@/\n"
+                + "                                     ,#***************/&%(/*****////***/%@@@@@%.\n"
+                + "                                     ,(.      **       %@@@@&&&@@@@@&&&@@&&@@@#.\n"
+                + "                                     ,#.       .*.     ,@@@@@%%&@@@&#&@&&@@@@#* .(&/\n"
+                + "                                     ,#.         *,  */ %@@@@(  /@@@%.    ,#&(*/    ,&%.\n"
+                + "                                     .#(*******,   %,   */      (@%&@&        *.#. ,/, .#/\n"
+                + "                                     .#,     (.     #(          #@&(@@&      *,  /*    ./*#%\n"
+                + "                                     .#,  ./.       .*.(.      .&@@/%@@%    *,    **      (.#/\n"
+                + "                                     .(,  ,/.        .*  ,/    *&@@//@@@(  /.      *,     (  *#\n"
+                + "                                     .#,     */       ./   .(. /@@@#,&@@@//         /.    (   .&.\n"
+                + "                                     .#/,      ./      ,*     ,%@@@%.#@@@%.         .(   ./    .&.\n"
+                + "                                     .#*.,.    .,.(     /.    .#@@@&**@@@@(          **  *,     /(\n"
+                + "                                     .%*.,     .,  ./   .*    .%@@@@#.&@@@@,         (  /     .#.\n";
+
+        System.out.println(face);
+
+
+        // Welcome statement
+        System.out.println(dashedLine + "Howdy partner! My name is Doug Dimmadome.\n"
+                + "Now what can I do for ya?\n" + dashedLine);
+    }
+
+
+    public static void main(String[] args) {
+
+        sayWelcome();
+        Scanner input = new Scanner(System.in);
+
+        while (true) {
+
+            String command = input.nextLine().trim();
+
+            if (command.equals("bye")) {
+                break;
+            } else if (command.equals("list")){
+                doList();
+            } else if (command.startsWith("mark")) {
+                doMark(command);
+            } else if (command.startsWith("unmark")) {
+                doUnmark(command);
+            } else if (command.startsWith("todo")) {
+                if (counter >= 100) {
+                    System.out.println(dashedLine + "Sorry partner, the list is full!" + dashedLine);
+                    continue;
+                }
+                doToDo(command);
+            } else if (command.startsWith("deadline")) {
+                if (counter >= 100) {
```

Comment:
Max. number of tasks should be labelled as a constant instead of using a magic number (100) here